### PR TITLE
docker: build image that runs in a docker env.

### PR DESCRIPTION
**Pull request by greut**
goreleaser don't like CGO_ENABLED=1

to be tested...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+VCS_REF="$(shell git rev-parse HEAD)"
+
+BUILD_DATE="$(shell date -u +"%Y-%m-%dT%H:%m:%SZ")" 
+
+docker-build:
+	docker build -t exo .
+	docker tag exo $(DOCKER_ID_USER)/exo
+	docker push $(DOCKER_ID_USER)/exo
```
Comment (greut) [inline Makefile]: 1. no push
2. you'll have to use the `--build-arg`
3. is the cli `main.version` ready?

Comment (pierre-emmanuelJ) [reply Makefile]: okay thx

File: Dockerfile [hunk 2]
```diff
@@ -4,14 +4,20 @@ ADD . /src
...
 ARG VCS_REF
-ENV ENV_VCS_REF=$VCS_REF
-
 ARG BUILD_DATE
-ENV ENV_BUILD_DATE=$BUILD_DATE
 
 ENV CGO_ENABLED=1
 RUN go build -mod vendor -o exo \
-        -ldflags "-s -w -X main.version=${ENV_BUILD_DATE} -X main.commit=${ENV_VCS_REF}"
+        -ldflags "-s -w -X main.version=${BUILD_DATE} -X main.commit=${VCS_REF}"
```
Comment (greut) [inline Dockerfile]: version <-> the git tag.

Comment (pierre-emmanuelJ) [reply Dockerfile]: 👍

**Pull request merged**